py/client/tests/test_session.py

```diff
@@ -13,7 +13,6 @@
 from pydeephaven.session import SharedTicket
 from tests.testbase import BaseTestCase
 
-
```

Comment:
this newline should be restored, py convention is to have two newlines before top-level funcs/classes

Reply:
Done